packages/admin-ui/src/switch/switch.style.ts

```diff
@@ -0,0 +1,127 @@
+import { colors, focusVisible } from '@vtex/admin-ui-core'
+import type { StyleProp } from '@vtex/admin-ui-core'
+import { get } from '@vtex/admin-ui-util'
+
+const css = (csx: StyleProp) => csx
+
+const green40 = get(colors, 'green40')
+const green50 = get(colors, 'green50')
+const gray40 = get(colors, 'gray40')
+
+// State styles
+const active = css({ bg: green40 })
+const inactive = css({
+  height: '1.25rem',
+  width: '2.25rem',
+  top: '0.06rem',
+  left: '0.06rem',
+  marginRight: '0.125rem',
+})
+
+const hoverInactive = css({ bg: gray40 })
+const hoverActive = css({
+  bg: green50,
+})
+
+const trackDefault = css({
+  bg: '$form.neutralInactive',
+})
+
+const focusActive = css({ '&:hover': hoverActive })
+const focusInactive = css({ '&:hover': hoverInactive })
+
+// Element styles
+const thumbDimensions = css({
+  '&:after': {
+    width: '0.75rem',
+    height: '0.75rem',
+    margin: '0.25rem',
+  },
+
+  '&:checked': {
+    '&:after': {
+      transform: 'translateX(1rem)',
+    },
+  },
+
+  '&:disabled': {
+    '&:after': {
+      bg: (theme) => get(theme, 'fg.disabled', ''),
+      margin: '0.2rem',
+    },
+  },
+})
+
+export const thumb = css({
+  bg: (theme) => get(theme, 'fg.form.neutralChecked'),
+  content: '""',
+  display: 'block',
+  position: 'absolute',
+  borderRadius: '1.25rem',
+  transition: 'transform .25s ease',
+})
```

Comment:
Is this being used?

Reply:
Yep! On the [`switch-button.tsx` file](https://github.com/vtex/admin-ui/pull/162/files#diff-57116421a4ae120096441bace8c7ccf94890bd9c19648d0abb2554fd7b3e0738R17).